spec/models/user_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.') }
```

Comment:
Thank you for your dedication and commitment into comprehensively testing the models and their validations. 🚀 

[OPTIONAL] To ensure that all your models are fully tested as per project requirements, all public methods should be tested. It would be best to include tests for:

- User model:  `most_recent_post` method which should check if it returns 3 most recent posts
- Post model: `most_recent_comments` method which should check if it returns 5 most recent comments per post
- `update_comment_counter` and `update_like_counter` which should check if creating comments and likes updates the counters when the methods are invoked.

### Summary

🎯 Overall, you have displayed great knowledge in testing different validations.